.github/workflows/ci_build.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
       # fail-fast is true by default
       fail-fast: false
       matrix:
-        hardware: ["cuda", "cuda-trtllm", "rocm", "intel-xpu", "intel-cpu"]
+        hardware: ["cuda", "rocm", "intel-xpu", "intel-cpu"]
```

Comment:
This removes it.

Reply:
Would this make more sens moving forward? 👍🏻 👎🏻 https://github.com/huggingface/text-generation-inference/pull/2916/files